libsolidity/codegen/ir/IRGenerator.h

```diff
@@ -87,10 +87,10 @@ class IRGenerator
 	/// Generates code that assigns the initial value of the respective type.
 	std::string generateInitialAssignment(VariableDeclaration const& _varDecl);
 
-	/// Generates implicit constructors for all contracts in the inheritance hierarchy of
+	/// Generates constructors for all contracts in the inheritance hierarchy of
 	/// @a _contract
-	/// If there are user defined constructors, their body will be included in implicit constructors body.
-	void generateImplicitConstructors(ContractDefinition const& _contract);
+	/// If there are user defined constructors', their body will be included in the implicit constructors body.
```

Comment:
```suggestion
	/// If there are user defined constructors, their body will be included in the implicit constructors body.
```

Reply:
I looked it up now :P http://www.sussex.ac.uk/informatics/punctuation/apostrophe/possessives
and it seems the suggestion here is correct.